digiwf-integrations/digiwf-verification-integration/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>digiwf-verification-integration</artifactId>
+    <packaging>pom</packaging>
+    <version>0.13.0-SNAPSHOT</version>
```

Comment:
Use parent version

Reply:
when I use project.version, I get maven errors. 
I think that it is no big problem since we are using maven-versions-plugin.

Reply:
Cant you just remove the version here, then the parent version is used?

Reply:
I have to enter a version: 
![image](https://user-images.githubusercontent.com/115698474/201867734-f6e958af-c184-4d89-9024-27f7ea56b8f8.png)